walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -223,6 +284,34 @@ public void justEnoughRoom() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void notEnoughRoomForOptionalAttendee() {
+    // Add an optional attendee, but make it so that there is not enough time for the meeting
+    // if they attend. 
+    //
+    // Events  : |--A--|     |----A----|
+    //                    |C|    
```

Comment:
Should be B and shift a little bit to the left to align with the end time of first event of A.